### DISCUSSION TITLE
Validation Error - Arming Requires Code - Just started recently

### DISCUSSION THREAD

**Discussion by vipul128** [tsightler/ring-mqtt#850] [Support Q&A]
I've been using the ring-mqtt adding for a while now, probably over a year and all of a sudden about a week or 2 ago I started getting this error that arming requires a code.  Again, I've never had this error before and I've been able to arm home and away without issues for the past 1+ years.  I've also not changed any settings on my actual Ring account either.  Is there something I'm missing or something that has changed recently?

EDIT: I should add that disarming still works just fine using this integration.

Failed to call service alarm_control_panel/alarm_arm_home. Validation error: Arming requires a code but none was given for alarm_control_panel.<name of ring alarm>

**Comment by tsightler** [comment]
Duplicate of https://github.com/tsightler/ring-mqtt/issues/846, https://github.com/tsightler/ring-mqtt/discussions/847 & https://github.com/tsightler/ring-mqtt/discussions/843, please use search prior to posting.

The ring-mqtt 5.6.5 version, released today, has a workaround for this HA broken behavior.

**Comment by zhangbin088** [comment]
alarm_control_panel:
    - unique_id: alarm001 
      name: "panel"
      supported_features:
        - arm_away 
      state_topic: "alarmdecoder/panel"
      value_template: "{{value_json.state}}"
      command_topic: "alarmdecoder/panel/set"
      code_arm_required: false

**Comment by zhangbin088** [comment]
code_arm_required：false
